applications/directory/src/components/ProfileReview.js

```diff
@@ -53,14 +57,40 @@ const ProfileReview = ({ onSubmit }) => {
       })
     }),
     onSubmit: async (vals) => {
-      await onSubmit(vals);
-      resetForm();
-      // TODO - navigate to thank you page/review complete
+      try {
+        await onSubmit(vals);
+        router.push('/reviews/review-complete');
+        resetForm();
+        setSubmissionError('');
+      } catch (error) {
+        setSubmissionError(
+          'An unexpected error occurred. Please try again to submit your review shortly.'
+        );
+
+        alertElement.current.scrollIntoView({
+          behavior: 'smooth',
+          block: 'center'
+        });
+      }
     }
   });
 
   return (
     <Box>
+      {submissionError && (
+        <Alert
+          ref={alertElement}
+          id={'#alert'}
```

Comment:
Why `id={'#alert'}` instead of `id="alert"`?

Reply:
It probably got there by mistake. It does not need id at all. Deleting it